README.adoc

```diff
@@ -256,7 +255,7 @@ cd start
 mvn liberty:start-server
 ----
 
-After the server is running, you can find your new Hypermedia-driven `inventory` service at the following URL:
+After the server runs, you can find your new hypermedia-driven `inventory` service at the following URL:
```

Comment:
Maybe we should change the starting to be "Once the server is running, ..."?

Reply:
How about "After the server is running,"...? Our grammar checker prefers "once" to mean "one time" instead of "after" or "when."

Reply:
We decide to keep the original.